File: 04. Sort 0s 1s and 2s/sortList.py
size = int(input("Enter the nunmber of elements: \n"))

listOfElements = list(map(int,input("Enter the elements separated by space: \n").strip().split()))
zeroCount=0
oneCount=0
twoCount=0
for item in listOfElements:
    if(item == 0):
        zeroCount+=1
    elif(item == 1):
        oneCount+=1
    else:
        twoCount+=1
listOfElements.clear()
for x in range(zeroCount):
    listOfElements.append(0)
for x in range(oneCount):
    listOfElements.append(1)
for x in range(twoCount):
    listOfElements.append(2)

    
print("Sorted List : {}\n".format(listOfElements))

    

File: 04. Sort 0s 1s and 2s/sortList.java
import java.io.*;
import java.util.Arrays;

class sortList
{
    public static void main(String args[])throws IOException
    {
        BufferedReader br =new BufferedReader(new InputStreamReader(System.in));
        System.out.println("Enter the number of Elements:");
        int size = Integer.parseInt(br.readLine());
        int listOfElements[] = new int[size];
        int zeroCount = 0, oneCount = 0, twoCount = 0;
        System.out.println("Enter the Elements:");     
        
        String[] strs = br.readLine().trim().split("\\s+");
      
        for(int i=0;i<size;i++)
        {
            listOfElements[i] = Integer.parseInt(strs[i]);
        }

    
        for(int i=0;i<size;i++)
        {
            if(listOfElements[i] == 0)
                zeroCount++;
            else if(listOfElements[i] == 1)
                oneCount++;
            else
                twoCount++;
        }
        System.out.println(oneCount);
        System.out.println(zeroCount);
        System.out.println(twoCount);
        for(int i = 0;i<size;i++)
        {
            if(zeroCount >= 1)
            {
                listOfElements[i]=0;
                zeroCount--;
            }
            else if(oneCount >= 1)
            {
                listOfElements[i]=1;
                oneCount--;
            }
            else if(twoCount >= 1)
            {
                listOfElements[i]=2;
                twoCount--;
            }
        }
        
        for(int i = 0;i < size;i++)
        {
            System.out.print(listOfElements[i]+" ");
            
        }
        
       
    }
}